### ISSUE TITLE
@types/jqlite: Minor version broke compatibility with @types/jquery

### ISSUE THREAD

**Issue by douglas-treadwell** [DefinitelyTyped/DefinitelyTyped#31733]
This commit created a TypeScript error when used together with jQuery.

https://github.com/DefinitelyTyped/DefinitelyTyped/commit/45903423f3d11c85f16a83771a85b1312dbb9350#diff-fa0930f1e24e0602511bb3207034d60c

```
node_modules/@types/jquery/index.d.ts:8186:5 - error TS2717: Subsequent property declarations must have the same type.  Property 'char' must be of type 'string', but here has type 'any'.

8186     char: any;
         ~~~~


node_modules/@types/jquery/index.d.ts:8188:5 - error TS2717: Subsequent property declarations must have the same type.  Property 'key' must be of type 'string', but here has type 'any'.

8188     key: any;
````

**Comment by douglas-treadwell** [comment]
Note: this issue was due to @types/angular-mocks having a dependency on @types/angular version "*", which meant that @types/angular-mocks caused angular to be installed with a new minor version, so this probably isn't a bug for jqlite or jquery.

**Comment by matthiasdailey-ccri** [comment]
You can install the version of `@types/jquery` before [the commit that changed `char` from `any` to `string`](https://github.com/DefinitelyTyped/DefinitelyTyped/commit/823068c14c1e9266b1f80b48f83effb7ecfb07a2), which was published in version 3.3.28.
```
"@types/jquery": "<3.3.28",
```

**Comment by Fundebug** [comment]
I got the same error when using AngularJS 1. Solved by install @types/angular 1.6.52
